x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -415,7 +415,7 @@ export type PolicyInfo = Immutable<{
 
 export type HostInfo = Immutable<{
   metadata: HostMetadata;
-  host_status: HostStatus;
+  host_status?: HostStatus;
```

Comment:
Why was this made optional? Because agent-enrichment inside `getHostData` is now optional?

The reasoning behind this being non-optional in the first place was a tacit guarantee for the endpoint specific pages that this field would always be populated (unless errs).

Reply:
Given that we decided to add getHostInfo, I'll revert the change here.